ranking/get_common_data.go

```diff
@@ -0,0 +1,56 @@
+package ranking
+
+import (
+	"github.com/PretendoNetwork/nex-go"
+	ranking "github.com/PretendoNetwork/nex-protocols-go/ranking"
+)
+
+func getCommonData(err error, client *nex.Client, callID uint32, uniqueID uint64) uint32 {
+	if commonRankingProtocol.getCommonDataHandler == nil {
+		logger.Warning("Ranking::GetCommonData missing GetCommonDataHandler!")
+		return nex.Errors.Core.NotImplemented
+	}
+	rmcResponse := nex.NewRMCResponse(ranking.ProtocolID, callID)
+	server := client.Server()
+
+	if err != nil {
+		logger.Error(err.Error())
+		rmcResponse.SetError(nex.Errors.Ranking.Unknown)
+	}
+
+	commonDataErr, commonData := commonRankingProtocol.getCommonDataHandler(uniqueID)
+
+	if commonDataErr == nil {
+		rmcResponseStream := nex.NewStreamOut(server)
+
+		rmcResponseStream.WriteBuffer(commonData)
+
+		rmcResponseBody := rmcResponseStream.Bytes()
+
+		rmcResponse.SetSuccess(ranking.MethodGetCommonData, rmcResponseBody)
+	}
```

Comment:
I've noticed a couple places like this where `err` is only being checked if it *is* `nil`, and doing a success. But there is nothing being done for if it *isn't* `nil`

This should probably throw `Ranking::NotFound` if `commonDataErr != nil`. That way we follow "happy path", where the default flow is a success and these kinds of checks would be for errors and to bail early